specificatie/openapi.yaml

```diff
@@ -1271,115 +1219,52 @@ components:
-          title: "Huisnummer"
-          description: "Een door of namens het bevoegd gemeentelijk orgaan ten aanzien\
-            \ van een adresseerbaar object toegekende nummering. Alle natuurlijke\
-            \ getallen tussen 1 en 99999."
-          maximum: 99999
-          example: 23
-        huisnummertoevoeging:
-          type: "string"
-          title: "Huisnummertoevoeging"
-          description: "Een door of namens het bevoegd gemeentelijk orgaan ten aanzien\
-            \ van een adresseerbaar object toegekende nadere toevoeging aan een huisnummer\
-            \ of een combinatie van huisnummer en huisletter. a - z , A - Z , 0 –\
-            \ 9"
-          pattern: "^[a-zA-Z0-9]{1,4}$"
-          maxLength: 4
-          example: "IV"
-        aanduidingBijHuisnummer:
-          $ref: "#/components/schemas/AanduidingBijHuisnummer_enum"
-        identificatiecodeNummeraanduiding:
-          type: "string"
-          title: "identificatiecodeNummeraanduiding"
-          description: "De unieke aanduiding van een NUMMERAANDUIDING. Combinatie\
-            \ van de viercijferige 'gemeentecode' de tweecijferige 'objecttypecode'\
-            \ en een voor het betreffende objecttype binnen een gemeente uniek tiencijferig\
-            \ 'objectvolgnummer'. De objecttypecode kent in de BAG de volgende waarde:20\
-            \ nummeraanduiding. Combinatie van de viercijferige 'gemeentecode' (volgens\
-            \ GBA tabel 33, Gemeententabel), de tweecijferige 'objecttypecode' en\
-            \ een voor het betreffende objecttype binnen een gemeente uniek tiencijferig\
-            \ 'objectvolgnummer'. De objecttypecode kent in de BAG de volgende waarde:20\
-            \ nummeraanduiding."
-          maxLength: 16
-          example: "0518200000366054"
-        naamOpenbareRuimte:
-          type: "string"
-          title: "Naam openbare ruimte"
-          description: "Een door het bevoegde gemeentelijke orgaan aan een OPENBARE\
-            \ RUIMTE toegekende benaming Tekens gecodeerd volgens de UTF-8 standaard"
-          maxLength: 80
-          example: "Loosduinsekade"
-        postcode:
-          type: "string"
-          title: "Postcode"
-          description: "De door PostNL vastgestelde code behorende bij een bepaalde\
-            \ combinatie van een naam van een woonplaats, naam van een openbare ruimte\
-            \ en een huisnummer"
-          pattern: "^[1-9]{1}[0-9]{3}[A-Z]{2}$"
-          example: "2571CC"
-        woonplaatsnaam:
-          type: "string"
-          title: "Woonplaatsnaam"
-          description: "De door het bevoegde gemeentelijke orgaan aan een WOONPLAATS\
-            \ toegekende benaming. Tekens gecodeerd volgens de UTF-8 standaard."
-          maxLength: 80
-          example: "Utrecht"
+      allOf:
+       - $ref : "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/v1.1.0/specificatie/openapi.yaml#/components/schemas/Adres"
+       - properties:
```

Comment:
naamOpenbareRuimte ontbreekt

Reply:
waarom heb je deze component gemaakt?

- voor hergebruik in bewoning zou ik ook adresseerbaarObjectIdentificatie hierin opnemen, maar geen functieAdres
- voor hergebruik in verblijfplaatshistorie kun je veel meer opnemen, zoals indicatieVestigingVanuitBuitenland, locatiebeschrijving, vanuitVertrokkenOnbekendWaarheen, enz.

Ik zou dus voorstellen om deze in de basis voor zowel verblijfplaatshistorie als voor bewoning herbruikbaar te maken, door functieadres eruit te verwijderen door naamOpenbareRuimte en adresseerbaarObjectIdentificatie eraan toe te voegen.

Vervolgens kan je nog kijken of je ook nog een component maakt op basis van binnenlandsAdres(basis) die voor zowel verblijfplaats als verblijfplaatshistorie herbruikbaar is.